src/util/images.tsx

```diff
@@ -48,9 +48,17 @@ export const localLxdToRemoteImage = (image: LxdImage): RemoteImage => {
     arch: image.architecture === "x86_64" ? "amd64" : image.architecture,
     os: image.properties?.os ?? "",
     created_at: new Date(image.uploaded_at).getTime(),
-    release: image.properties?.release ?? "",
-    server: LOCAL_IMAGE,
+    release:
+      image.properties?.os === "ubuntu"
+        ? (image.properties.version ?? "")
+        : (image.properties?.release ?? ""),
+    server: image.cached ? image.update_source?.server : LOCAL_IMAGE,
     type: image.type,
+    cached: image.cached,
+    variant:
+      image.properties?.os === "ubuntu"
+        ? image.properties?.release
+        : image.properties?.variant,
```

Comment:
```suggestion
```
Seems not needed to me. Can it be removed?

Reply:
closing this since the discussion is happening on the other comment.